roles/osp_observability/molecule/default/molecule.yml

```diff
@@ -0,0 +1,18 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: podman
+platforms:
+  - name: instance
+    image: docker.io/pycontribs/centos:8
```

Comment:
Would this be running in downstream testing at some point? It might be worth either pointing at the quay.io mirror (if such a thing exists) or mirroring this to the quay.io/infrawatch/ organization as I've done in the past to make sure the test doesn't fail on pulling images from docker.io.

If this will only run upstream in GH Actions, then probably not a problem? Not sure if GH Actions has the same issue with pulling images from docker.io or not and hitting the cap limit.

Reply:
Good catch! pycontribs DOES exist in quay https://quay.io/repository/pycontribs/centos The tags are a bit different and it doesn't look like it's getting updates, but I'll swap it for now to avoid problems. I'll also try to find something that might stay more up to date.

Reply:
So neither image is getting updates it seems. This would work on ubi8, but it doesn't come with yamllint by default and I didn't feel like adding additional testing tasks at this point. I think this will do fine for now.